saleor/graphql/order/types.py

```diff
@@ -548,7 +754,7 @@ def resolve_lines(root: models.Order, info):
     @staticmethod
     @permission_required(OrderPermissions.MANAGE_ORDERS)
     def resolve_events(root: models.Order, _info):
-        return root.events.all().order_by("pk")
+        return root.events.prefetch_related("user").all().order_by("pk")
```

Comment:
Ideally, the `OrderEvent` type should have a data loader for users, I think the simplest one would do the job:
```
class UserByIdLoader(DataLoader):
    context_key = "user_by_id"

    def batch_load(self, keys):
        users = User.objects.in_bulk(keys)
        return [users.get(user_id) for user_id in keys]
```

Reply:
We already discovered the issue with the missing dataloader here and in OrderEvent type.  I believe @fowczarek already created the tasks for this as the changes require more work to do it properly. . I've added here the prefetch as I discover that we are generating huge amount of queries by calling order events. But the loaders here should be added in separate PR.